spec/circle_count_graph_spec.rb

```diff
@@ -40,11 +40,9 @@
       end
     end
     context 'one circle' do
-      let(:data) { { red: 1 } }
       include_examples 'has a width and height of', 20, 20
```

Comment:
I removed also the let(:data) { { red: 1 } } because it's defined in the global settings and thus also redundant!